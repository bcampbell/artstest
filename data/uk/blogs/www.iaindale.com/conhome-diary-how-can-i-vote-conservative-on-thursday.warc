WARC/1.0
Content-Length: 18269
Content-Type: application/http; msgtype=response
Warc-Date: 2014-05-16T17:18:39Z
Warc-Record-Id: urn:X-scrapeomat:1400260719569376807
Warc-Target-Uri: http://www.iaindale.com/posts/2014/05/16/conhome-diary-how-can-i-vote-conservative-on-thursday
Warc-Type: response
X-Scrapeomat-Srcurl: http://www.iaindale.com/posts/2014/05/16/conhome-diary-how-can-i-vote-conservative-on-thursday

HTTP/1.1 200 OK
Transfer-Encoding: chunked
Cache-Control: must-revalidate, private, max-age=0
Connection: keep-alive
Content-Type: text/html; charset=utf-8
Date: Fri, 16 May 2014 17:18:39 GMT
Etag: "5f54631bd0269f6e47a05c0ee6560110"
Server: nginx/1.4.5
Status: 200 OK
X-Rack-Cache: miss
X-Request-Id: 40aab6ce84eeaef6a4142625dd3e14a6
X-Runtime: 0.042805
X-Ua-Compatible: IE=Edge,chrome=1

45bd
<!DOCTYPE html>
<html lang='en'>
<head>
<meta charset='utf-8'>
<meta content='IE=Edge,chrome=1' http-equiv='X-UA-Compatible'>
<meta content='width=device-width, initial-scale=1.0' name='viewport'>
<meta content='/assets/apple-touch-icon-144x144-9e191be19deaf4db925fbd1e4c38041b.png' property='og:image'>
<link href='http://www.iaindale.com/feed' rel='alternate' title='RSS feed' type='application/rss+xml'>
<title>ConHome Diary: How Can I Vote Conservative on Thursday? - Iain Dale</title>
<meta content="authenticity_token" name="csrf-param" />
<meta content="ftMTgtsGvES+tgKu2G2ij/t/ky1kCvZNtNUx+UkxP80=" name="csrf-token" />
<!-- Le HTML5 shim, for IE6-8 support of HTML elements -->
<!--[if lt IE 9]>
<script src="http://html5shim.googlecode.com/svn/trunk/html5.js" type="text/javascript"></script>
<![endif]-->
<link href="/assets/application-1d2c27875810ee7e17403de2fda52f49.css" media="all" rel="stylesheet" type="text/css" />
<script src="/assets/application-77ae26275b159e860ff4ff14d96f462f.js" type="text/javascript"></script>
<script type="text/javascript" src="//use.typekit.net/plw5ree.js"></script>
<script type="text/javascript">try{Typekit.load();}catch(e){}</script>
<link href='assets/apple-touch-icon-144x144.png' rel='apple-touch-icon-precomposed' sizes='144x144'>
<link href='assets/apple-touch-icon-114x114.png' rel='apple-touch-icon-precomposed' sizes='114x114'>
<link href='assets/apple-touch-icon-72x72.png' rel='apple-touch-icon-precomposed' sizes='72x72'>
<link href='assets/favicon.ico' rel='shortcut icon'>
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-497280-12']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>
</head>
<body class='with-notice'>
<div id='fb-root'></div>
<script type="text/javascript" src="https://connect.facebook.net/en_US/all.js"></script>
<script type="text/javascript">
/* <![CDATA[ */
FB.init({appId:"169056269900163", status: "false", cookie: "false", xfbml: "false"});

/* ]]> */
</script>

<div class='navbar navbar-fixed-top'>
<div class='notice-bar'>
<div class='container'>
<p class='hidden-phone'>
This web site uses cookies to improve your experience.  I&rsquo;ll assume you&rsquo;re ok with that. <a href="/cookies">More info</a>.
<a href="http://www.iaindale.com/cookies/dismiss_notice" class="close" data-method="put" rel="nofollow">&times;</a>
</p>
<p class='visible-phone'>
This web site uses <a href="/cookies">cookies</a>.
<a href="http://www.iaindale.com/cookies/dismiss_notice" class="close" data-method="put" rel="nofollow">&times;</a>
</p>
</div>
</div>
<div class='navbar-inner'>
<div class='container'>
<!-- .btn-navbar is used as the toggle for collapsed navbar content -->
<a class='btn btn-navbar' data-target='.nav-collapse' data-toggle='collapse'>
<span class='icon-bar'></span>
<span class='icon-bar'></span>
<span class='icon-bar'></span>
</a>
<!-- Be sure to leave the brand out there if you want it shown -->
<a class='brand' href='http://www.iaindale.com/'>
<img alt="Logo" src="/assets/logo-59b523a4320820a7b29dcf8043ca1c45.png" />
</a>
<!-- Everything you want hidden at 940px or less, place within here -->
<div class='nav-collapse collapse'>
<div class='nav pull-left'>
<li>
<a href='/'>
<i class='icon-pencil'></i>
Posts
</a>
</li>
<li>
<a href='/interviews'>
<i class='icon-headphones'></i>
Radio
</a>
</li>
<li>
<a href='/about'>
<i class='icon-info-sign'></i>
About
</a>
</li>
<li>
<a href='/contact'>
<i class='icon-envelope'></i>
Contact
</a>
</li>
</div>
<ul class='nav nav-social pull-right visible-desktop'>
<li>
<a data-placement='bottom' href='https://twitter.com/IainDale' rel='tooltip' title='Follow me'>
<i class='icon-twitter'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.flickr.com/photos/iaindale/' rel='tooltip' title='Flickr photostream'>
<i class='icon-flickr'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.youtube.com/user/iaindale' rel='tooltip' title='YouTube channel'>
<i class='icon-youtube'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.iaindale.com/feed' rel='tooltip' title='RSS feed'>
<i class='icon-rss'></i>
</a>
</li>
</ul>
</div>
</div>
</div>
</div>
<div class='live-show hidden-phone'>
<div class='container'>
<div class='row'>
<div class='span9'>
<h3>Listen to Iain Dale live on LBC. On today&rsquo;s show...</h3>
<ul>
	<li>4.00pm Women Bishops: How can women bishops change the C of E?</li>
	<li>4.30pm Stuart Hall verdict report</li>
	<li>4.45pm Who is Narenda Modi, the new Indian Prime Minister?</li>
	<li>5.00pm Climate Change: Is another Climategate scandal about to unfold?</li>
	<li>5.35pm Business with Ben Kumar</li>
	<li>5.40pm Why is the Chilcott report being delayed for so long?</li>
	<li>5.45pm Nigel Farage &amp; James O’Brien With Danny Finkelstein</li>
	<li>6.00pm International Day Against Homophobia – Is Homophobia getting worse?</li>
	<li>6.35pm Business with Ben Kumar</li>
	<li>6.45pm Kenya update</li>
	<li>7.00pm How did your parents interfere in your love life?</li>
	<li>7.30pm Colleen Nolan interview</li>
</ul>
<a class='btn live-player' href='http://ukrp.musicradio.com/lbc973/live'>
<i class='icon-headphones'></i>
Listen now
</a>
<img alt="Lbc-logo-small" src="/assets/lbc-logo-small-3f9f7867c8ddbe34bde4a820469f6510.png" />
</div>
</div>
</div>
</div>
<div class='container content-wrapper'>

<div class='row'>
<div class='span7'>
<div id='content'>
<div class='post' id='post-53752e7ebd137dc1a40a2fbc'>
<div class='post-content'>
<p class='category'>Diary</p>
<h1>ConHome Diary: How Can I Vote Conservative on Thursday?</h1>
<p>
<small class='muted'>16 May 2014 at 14:14</small>
</p>
<div class='body'>
<p>I face a real dilemma. I opened my postal vote yesterday but I just can’t bring myself to vote Conservative. The reason is simply that Marta Andreassen is the fourth placed candidate on the ballot paper for the Tories. And one place above her is Richard Ashworth, a lovely man, but so far as I can determine, a complete Europhile. Luckily, I have another option, to vote in the Eastern Region. But what do I find there? Below the very excellent Vicky Ford on the list is Geoffrey Van Orden, a man known to East Anglian Tories as Geoffrey Van Ordinaire. And then below him is the ultimate re-ratter David Campbell-Bannerman. What’s a boy to do? Spoil his ballot paper? Vote <span class="caps">UKIP</span>? I suppose in the end I will hold my nose and do the “right” thing, but if I do, it will be Dan Hannan or Vicky Ford who gets my vote. Not the others. OK, I know that’s not how it works, but it’s the only way I can justify what I am about to do. <br />
<b>*<br />
All of this points to the fact that a future Conservative government should change the voting system for future European elections. If we have to have PR, let’s at least have a form of PR where the candidate matters. Let’s change to a system of PR where, if there are 8 seats in a region, we can allocate the 8 votes to the candidates of our choice. Yes, we could vote for candidates from different parties, but what’s so wrong with that? It would allow people in my South East region to vote for Dan Hannan, Nigel Farage and Diane James. And I suspect that would be the choice of many, many Tories.<br />
*</b><br />
I once spent three hours with Nick Griffin, interviewing him for a magazine piece. It was a fascinating experience because after the first twenty minutes I realised that there was nothing to him. He was an empty vessel. Once you got beyond his normal subjects of race and immigration there was literally nothing there. Ask him about schools or the health service and he just couldn’t answer the question. And this is where broadcast media interviewers go wrong with him. They play into his hands and quote ridiculous things he has come out with from 1984 or 1996 and they then spend a few minutes going back and forth until they have done their requisite four minutes. And the audience is left none the wiser. Evan Davis did exactly that on Today on Wednesday. It was a waste of four minutes. I’ve decided I’ll have Griffin on my programme in the next few days too. I will, however, try to at least let the listeners make their own minds up about him and conduct the interview just how I would with any other party leader, no matter how repulsive I might personally find them. <br />
<strong>*</strong><br />
It’s rare that a battle for a select committee chairmanship is anything other than incredibly tedious. Not for the Defence Select Committee. The fight to replace James Arbuthnot has been fascinating to watch with several of the candidates using some very black arts indeed to discredit their opponents. There seemed to be so many contestants that I rather lost count of who remained in the battle, but from memory they were James Gray, Bob Stewart, Keith Simpson, Julian Lewis, Julian Brazier and Rory Stewart. Have I missed anyone? I have so far spoken to three MPs who have said that although they signed the nomination papers of one candidate, they would be voting for another. Who said Tory MPs were the most duplicitous electorate in modern Christendom?</p>
</div>
<div class='post-share'>
<h3>Share:</h3>
<ul>
<li>
<a class='fb_share' href='#' rel='tooltip' title='Share on Facebook'>
<i class='icon-facebook'></i>
</a>
<script type='text/javascript'>
$("#post-53752e7ebd137dc1a40a2fbc .fb_share").unbind("click.facebook_share").bind("click.facebook_share",function () {
  FB.ui({method: 'stream.publish', link: "http://www.iaindale.com/posts/2014/05/16/conhome-diary-how-can-i-vote-conservative-on-thursday", display: "popup", name: "ConHome Diary: How Can I Vote Conservative on Thursday?"});
  return false;

});

</script>
</li>
<li>
<a href="http://twitter.com/share?url=http%3A%2F%2Fwww.iaindale.com%2Fposts%2F2014%2F05%2F16%2Fconhome-diary-how-can-i-vote-conservative-on-thursday&amp;via=iaindale&amp;text=ConHome+Diary%3A+How+Can+I+Vote+Conservative+on+Thursday%3F&amp;related=&amp;count=vertical&amp;lang=en" class="tweet" rel="tooltip" title="Tweet"><i class='icon-twitter'></i></a>
</li>
</ul>
</div>

</div>
<div class='comments'>
<h2>1 comment</h2>
<p>
Sign up via
<a class='btn btn-mini btn-facebook' href='/auth/facebook'>
<i class='icon-facebook'></i>
Facebook
</a>
or
<a class='btn btn-mini btn-twitter' href='/auth/twitter'>
<i class='icon-twitter'></i>
Twitter
</a>
to comment.
</p>

<div class='comment' id='comment-53761770bd137dc1a40a4a8b'>
<div class='row-fluid'>
<div class='span1'>
<a href='http://twitter.com/Golfing_grannie'>
<img alt="Me0312_normal" src="http://pbs.twimg.com/profile_images/1961963016/Me0312_normal.jpg" />
</a>
</div>
<div class='span11'>
<a href="http://twitter.com/Golfing_grannie" class="name">Marcia Malia</a>
<time class="pull-right" data-time-ago="2014-05-16T14:49:36+01:00" datetime="2014-05-16T14:49:36+01:00" title="Fri, 16 May 2014 14:49:36 +0100">about 18 hours</time>
<div class='message'>
<p>Totally agree about the Euro elections. I was shocked to find that I couldn't vote for individuals.  It seems all wrong to me, and it is not in the tradition of the way we have always conducted elections in this country.  You can end up voting for people you have never heard of, because of one good one.</p>
</div>
<div class='replies'>

</div>
</div>
</div>
</div>

<div class='comment comment-form-wrapper'>
<p>
Sign up via
<a class='btn btn-mini btn-facebook' href='/auth/facebook'>
<i class='icon-facebook'></i>
Facebook
</a>
or
<a class='btn btn-mini btn-twitter' href='/auth/twitter'>
<i class='icon-twitter'></i>
Twitter
</a>
to comment.
</p>

</div>

</div>
</div>

</div>
</div>
<div class='span5 sidebar'>
<div class='profile-mini'>
<div class='row-fluid'>
<div class='span3'>
<img alt="Iain-dale-mini" src="/assets/iain-dale-mini-bc9d0893bf164220314ce5b232ba9198.png" />
</div>
<div class='span9'>
<p>
Iain Dale presents Drive on LBC, 4-8pm weekdays, and is Managing Director of Biteback Publishing.
<a href="/about">More</a>
</p>
</div>
</div>
</div>
<div class='signup-box'>
<p>
Sign up via
<a class='btn btn-mini btn-facebook' href='/auth/facebook'>
<i class='icon-facebook'></i>
Facebook
</a>
or
<a class='btn btn-mini btn-twitter' href='/auth/twitter'>
<i class='icon-twitter'></i>
Twitter
</a>
to comment.
</p>

</div>
<div class='daley-dozen'>
<h4>The Daley Dozen</h4>
<ul>
<li>
<span class='number'>1.</span>
<a href="http://www.drm.org/?p=2950">The Power of Radio</a> | <em>Robin Lustig</em>
</li>
<li>
<span class='number'>2.</span>
<a href="http://www.telegraph.co.uk/women/womens-life/10711537/10-things-no-one-tells-you-about-being-female-radio-presenter.html">10 Things You Need to Know About Talk Radio</a> | <em>Daily Telegraph</em>
</li>
<li>
<span class='number'>3.</span>
<a href="http://www.theguardian.com/tv-and-radio/2014/feb/14/katie-hopkins-interview">Brilliantly insightful Katy Hopkins Interview</a> | <em>The Guardian</em>
</li>
<li>
<span class='number'>4.</span>
<a href="http://blogs.telegraph.co.uk/news/peteroborne/100258499/ignore-the-london-media-sneers-anne-mcintoshs-tory-association-behaved-perfectly-properly-in-deselecting-her/">Deselections are Perfectly Proper</a> | <em>Peter Oborne</em>
</li>
<li>
<span class='number'>5.</span>
<a href="http://www.telegraph.co.uk/news/politics/conservative/10614852/Timothy-Yeos-deselection-Is-this-the-start-of-a-Tory-Spring.html">Deselections: Can They Lead to a Tory Spring?</a> | <em>Tim Bale</em>
</li>
<li>
<span class='number'>6.</span>
<a href="http://www.theguardian.com/media/2014/feb/03/tv-24-hour-news-channels-bbc-rolling?CMP=twt_gu">Do 24 Hour News Channels Have a Future?</a> | <em>The Guardian</em>
</li>
<li>
<span class='number'>7.</span>
<a href="http://www.ethosjournal.com/topics/politics/item/492-profile-sajid-javid">Profile of Sajid Javid by Iain Dale</a> | <em>Ethos Journal</em>
</li>
<li>
<span class='number'>8.</span>
<a href="http://www.theguardian.com/lifeandstyle/2013/dec/21/letter-to-son-cant-join-us-christmas">A Letter to My Son</a> | <em>The Guardian</em>
</li>
<li>
<span class='number'>9.</span>
<a href="http://www.theguardian.com/commentisfree/2013/sep/08/despise-politicians-my-part-murdoch-machine">Murdoch&#x27;s Part in My Downfall</a> | <em>Chris Huhne</em>
</li>
<li>
<span class='number'>10.</span>
<a href="http://blogs.telegraph.co.uk/news/danhodges/100233350/miliband-was-governed-by-narrow-political-interests-not-the-national-interests-or-those-of-syrian-children/">Why I Have Left Labour</a> | <em>Dan Hodges</em>
</li>
<li>
<span class='number'>11.</span>
<a href="http://blogs.telegraph.co.uk/news/iainmartin1/100233004/this-is-not-david-camerons-finest-hour/">Not Cameron&#x27;s Finest Hour</a> | <em>Iain Martin</em>
</li>
<li>
<span class='number'>12.</span>
<a href="http://votingandboating.blogspot.co.uk/2013/08/keeping-my-own-black-dog-on-lead.html">Mental Health: Keeping my &#x27;Black Dog&#x27; on a Lead</a> | <em>Andrew Kennedy</em>
</li>
</ul>
</div>
<div class='quote' id='post-52ef81441d1345d8f901dad3'>
<h4>Quote of the day</h4>
<blockquote>
<p>One of my favourite books is the Strange Death of Liberal England.</p>
<small>Michael Gove</small>
</blockquote>
<a href='/posts/2014/02/03/quote-of-the-day-michael-gove'>
<i class='icon-comments'></i>
0
</a>
<span class='sep'>/</span>
<a class='fb_share' href='#' rel='tooltip' title='Share on Facebook'>
<i class='icon-thumbs-up'></i>
Like
</a>
<script type='text/javascript'>
$("#post-52ef81441d1345d8f901dad3 .fb_share").unbind("click.facebook_share").bind("click.facebook_share",function () {
  FB.ui({method: 'stream.publish', link: "http://www.iaindale.com/posts/2014/02/03/quote-of-the-day-michael-gove", display: "popup", name: "Quote of the day: Michael Gove"});
  return false;

});

</script>
<span class='sep'>/</span>
<a href="http://twitter.com/share?url=http%3A%2F%2Fwww.iaindale.com%2Fposts%2F2014%2F02%2F03%2Fquote-of-the-day-michael-gove&amp;via=iaindale&amp;text=Quote+of+the+day%3A+Michael+Gove&amp;related=&amp;count=vertical&amp;lang=en" class="tweet" rel="tooltip" title="Tweet"><i class='icon-twitter'></i>Tweet</a>
</div>
<a class="twitter-timeline" data-dnt=true href="https://twitter.com/IainDale" data-widget-id="277899580778094592">Tweets by @IainDale</a>
<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src="//platform.twitter.com/widgets.js";fjs.parentNode.insertBefore(js,fjs);}}(document,"script","twitter-wjs");</script>
</div>
</div>
<div class='row'>
<div class='span7'>
<div class='footer'>
<p>
Copyright &copy; 2014 Iain Dale.  <a href="/terms">Privacy policy</a>.  <a href="/cookies">Cookies</a>.
<br>
Web site by <a href="mailto:russ@russellbrown.co.uk">Russell Brown</a>.
</p>
</div>
</div>
</div>
</div>
<div class='container visible-phone'>
<div class='navbar'>
<div class='navbar-inner'>
<ul class='nav nav-social'>
<li>
<a data-placement='bottom' href='https://twitter.com/IainDale' rel='tooltip' title='Follow me'>
<i class='icon-twitter'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.flickr.com/photos/iaindale/' rel='tooltip' title='Flickr photostream'>
<i class='icon-flickr'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.youtube.com/user/iaindale' rel='tooltip' title='YouTube channel'>
<i class='icon-youtube'></i>
</a>
</li>
<li>
<a data-placement='bottom' href='http://www.iaindale.com/feed' rel='tooltip' title='RSS feed'>
<i class='icon-rss'></i>
</a>
</li>
</ul>
</div>
</div>
</div>
</body>
</html>

0



