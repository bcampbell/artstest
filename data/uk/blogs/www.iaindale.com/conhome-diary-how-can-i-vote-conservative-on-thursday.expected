---
urls:
  - "http://www.iaindale.com/posts/2014/05/16/conhome-diary-how-can-i-vote-conservative-on-thursday"
canonical_url: 
headline: "ConHome Diary: How Can I Vote Conservative on Thursday?"
published: 2014-05-16T14:14
publication:
  - name: 
  - domain: www.iaindale.com
---
<div><p>I face a real dilemma. I opened my postal vote yesterday but I just can’t bring myself to vote Conservative. The reason is simply that Marta Andreassen is the fourth placed candidate on the ballot paper for the Tories. And one place above her is Richard Ashworth, a lovely man, but so far as I can determine, a complete Europhile. Luckily, I have another option, to vote in the Eastern Region. But what do I find there? Below the very excellent Vicky Ford on the list is Geoffrey Van Orden, a man known to East Anglian Tories as Geoffrey Van Ordinaire. And then below him is the ultimate re-ratter David Campbell-Bannerman. What’s a boy to do? Spoil his ballot paper? Vote<span>UKIP</span>? I suppose in the end I will hold my nose and do the “right” thing, but if I do, it will be Dan Hannan or Vicky Ford who gets my vote. Not the others. OK, I know that’s not how it works, but it’s the only way I can justify what I am about to do.<br/>
<b>*<br/>
All of this points to the fact that a future Conservative government should change the voting system for future European elections. If we have to have PR, let’s at least have a form of PR where the candidate matters. Let’s change to a system of PR where, if there are 8 seats in a region, we can allocate the 8 votes to the candidates of our choice. Yes, we could vote for candidates from different parties, but what’s so wrong with that? It would allow people in my South East region to vote for Dan Hannan, Nigel Farage and Diane James. And I suspect that would be the choice of many, many Tories.<br/>
*</b><br/>
I once spent three hours with Nick Griffin, interviewing him for a magazine piece. It was a fascinating experience because after the first twenty minutes I realised that there was nothing to him. He was an empty vessel. Once you got beyond his normal subjects of race and immigration there was literally nothing there. Ask him about schools or the health service and he just couldn’t answer the question. And this is where broadcast media interviewers go wrong with him. They play into his hands and quote ridiculous things he has come out with from 1984 or 1996 and they then spend a few minutes going back and forth until they have done their requisite four minutes. And the audience is left none the wiser. Evan Davis did exactly that on Today on Wednesday. It was a waste of four minutes. I’ve decided I’ll have Griffin on my programme in the next few days too. I will, however, try to at least let the listeners make their own minds up about him and conduct the interview just how I would with any other party leader, no matter how repulsive I might personally find them.<br/>
<strong>*</strong><br/>
It’s rare that a battle for a select committee chairmanship is anything other than incredibly tedious. Not for the Defence Select Committee. The fight to replace James Arbuthnot has been fascinating to watch with several of the candidates using some very black arts indeed to discredit their opponents. There seemed to be so many contestants that I rather lost count of who remained in the battle, but from memory they were James Gray, Bob Stewart, Keith Simpson, Julian Lewis, Julian Brazier and Rory Stewart. Have I missed anyone? I have so far spoken to three MPs who have said that although they signed the nomination papers of one candidate, they would be voting for another. Who said Tory MPs were the most duplicitous electorate in modern Christendom?</p>
</div>

